gui-libs/amtk/amtk-5.6.0.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome2 meson
+
+DESCRIPTION="Actions, Menus and Toolbars Kit for GTK+ applications"
+HOMEPAGE="https://wiki.gnome.org/Projects/Amtk https://gitlab.gnome.org/World/amtk"
+
+LICENSE="LGPL-2.1+"
+SLOT="5"
```

Comment:
Looks like the SOVERSION changed from 5.4.1 to 5.6.0:

````diff
-/usr/lib64/libamtk-5.so.0.1.0
+/usr/lib64/libamtk-5.so.1.0.1
````
I think we need a subslot and to update reverse dependencies to bind to it.